pycm/cm.py

```diff
@@ -631,6 +631,20 @@ def distance(self, metric):
                 TP=self.TP[i], FP=self.FP[i], FN=self.FN[i], TN=self.TN[i])
         return distance_dict
 
+    def dissimilarity_matrix(self):
+        """
+        Calculate dissimilarity matrix.
+
+        :return: dissimilarity matrix as dict
+        """
+        result = {class_name: dict(zip(self.classes, [0] * len(self.classes))) for class_name in self.classes}
+        matrix_array = self.to_array()
+        for class_index_1, class_name_1 in enumerate(self.classes):
+            for class_index_2, class_name_2 in enumerate(self.classes):
+                dist = int(sum(abs(matrix_array[class_index_1] - matrix_array[class_index_2])))
```

Comment:
so this is l1, right? are we planning to support l2 or other measures?

Reply:
Yes, we will support other metrics in future versions.